### ISSUE TITLE
[RFC] Should hover include :focus as well

### ISSUE THREAD

**Issue by zauan** [zionbuilder/zionbuilder#279]
Someone left a comment on the WordPress comments page for the plugin:

"While editing :hover styles (which should also include :focus by default because this is basic accessibility), the hover state should be reflected in the element. If I choose a color for a button, I want to see that updated live. Currently, you have to pick a color, close the color picker, then go hover over the element to see the change. Rinse and repeat until it’s the way you want."

This seems to be a valid idea and we've discussed this internally and came up with two ways for dealing with this:

1. Implement this by default, all :hover selectors to include :focus by default
2. Allow the user to create custom selector for element as this will give more control over the generated CSS as well as resolving this, however, the user should manually select :focus selector as long with the :focus selector

For anyone who has ideas regarding this matter, please don't hesitate to reply to this ticket.

regards,
Stefan

**Comment by swinggraphics** [comment]
Hi, I posted the issue on wp.org. :) It's really two issues in one, sorry. Limiting the discussion here to the :hover/:focus selector, I would say that I think the best UI would be for the menu to show ":hover, :focus" by default, but also let you select ":hover" and ":focus" separately.  This needs to be done for basic accessibility. Therefore, it is not a "Pro" feature, and should be standard. (It would be okay to limit setting them separately to Pro.)

**Comment by zauan** [comment]
Hi,

Yes, it was taken from WP.org. Here it would be easier to manage and discuss features and changes
